ifrs17-template/Test/ScenarioDataImportTest.ipynb

```diff
@@ -0,0 +1,868 @@
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await DataSource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(argsScenario);",
+                "\nvar ivsScenarioInitCashFlows = await DataSource.Query<IfrsVariable>().ToArrayAsync();",
+                "\nvar diffs = ivsScenarioInitCashFlows.Except(ivsBestEstimateInit, IfrsVariableComparer.Instance());"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "diffs.Where(x => actualEstimateTypes.Contains(x.EstimateType)).ToArray().Length.Should().Be(0);",
+                "\ndiffs.Where(x => !actualEstimateTypes.Contains(x.EstimateType)).ToArray().Length.Should().NotBe(0);",
+                "\ndiffs.Where(x => x.EstimateType == EstimateTypes.DA).ToArray().Length.Should().NotBe(0);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Scenario Init Actuals"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var actualsScenario = @\"",
+                "\n@@Main",
+                "\nReportingNode,Year,Month,Scenario",
+                "\nCH,2020,12,Test",
+                "\n@@Actual",
+                "\nDataNode,AocType,ValueType,AccidentYear,Value",
+                "\nDT1.1,CF,NIC,,-308\";"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await Import.FromString(actualsScenario).WithFormat(ImportFormats.Actual).WithTarget(DataSource).ExecuteAsync()"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await DataSource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(argsScenario);",
+                "\nvar ivsScenarioInitActuals = DataSource.Query<IfrsVariable>().ToArray();",
+                "\nvar diffs = ivsScenarioInitActuals.Except(ivsScenarioInitCashFlows, IfrsVariableComparer.Instance());"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "diffs.Where(x => actualEstimateTypes.Contains(x.EstimateType)).ToArray().Length.Should().NotBe(0);",
+                "\ndiffs.Where(x => x.EstimateType == EstimateTypes.CU).ToArray().Length.Should().Be(0);",
```

Comment:
why do you check only for CU? I would check for all EstimateType that are not related to Actuals - BE and Ra for example.

Reply:
The test section about CU was written to prove that the changes in Dimensions.csv from "DA,Deferrable Actuals,80,AoC,3,NotApplicable,,,,,," to "DA,Deferrable Actuals,80,AoC,7,NotApplicable,,,,,," was working properly. More specifically, we expected a chian of changes from CU => AM => DA. We also add C, L And F for completeness.  Testing all EstimateType not related to Actuals seems to me a bit too generic test. I would rather create a new issue to improve the test coverage in another notebook.

Reply:
proceed as discussed.